Library/Formula/cairo.rb

```diff
@@ -8,11 +8,9 @@ class Cairo < Formula
   keg_only :provided_pre_mountain_lion
 
   option :universal
-  # Tiger's X11 is simply way too old
-  option 'without-x', 'Build without X11 support' if MacOS.version > :tiger
 
   depends_on "pkg-config" => :build
-  depends_on :x11 => :optional if MacOS.version > :leopard
+  depends_on :x11 => if MacOS.version > :leopard then :optional else :recommended end
```

Comment:
I removed the `option 'without-x'` and I made it `:recommended` to preserve the old behaviour.
Then I extended it from just `:leopard` to both `:tiger` and `:leopard`.
I (rebased and) pushed a new patchset that should make this steps more visible.